### PR TITLE
impr(S3C-4034): Simplify soft limit

### PR DESCRIPTION
**Pull request by tmacro**

### CODE REVIEW

File: libV2/config/defaults.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
         "schedule": "0 0 0 * * 6"
     },
     "diskUsage": {
-        "expirationBlockSize": "24",
-        "mode": "local"
+        "retentionPeriod": 45,
```
Comment (jonathan-gramain) [inline libV2/config/defaults.json]: could be useful to have the unit type appear in the tunable name (so `retentionDays`)

**Review by vrancurel — APPROVED** [review]

File: libV2/tasks/DiskUsage.js [hunk 2]
```diff
@@ -184,75 +159,29 @@ class MonitorDiskUsage extends BaseTask {
...
-            }
+        let size = null;
+        try {
+            size = await this._getUsage();
+        } catch (error) {
+            moduleLogger.error(`error calculating disk usage for ${this._path}`, { error });
+            return;
         }
 
+        moduleLogger.info(`using ${formatDiskSize(size)}`, { usage: size });
```
Comment (jonathan-gramain) [inline libV2/tasks/DiskUsage.js]: I'd suggest adding a bit of context to this log message

**Comment by tmacro** [comment]
@jonathan-gramain  `warp 10 leveldb using`...  work for you?

**Review by jonathan-gramain — APPROVED** [review]

**Comment by jonathan-gramain** [comment]
@tmacro do you need to target `stabilization/7.9.0`? The ticket contains 7.9.0 in fixVersion.

**Review by anurag4DSB — APPROVED** [review]

**Pull request merged**